GetStream/stream-chat-android
# [3372] Add start/end padding customization support to MessageInputFie…

### 🎯 Goal

Add 'MessageInputFiledView' start/end padding customisation option

### 🛠 Implementation details

Added `stream_ui_message_input_padding_start` and `stream_ui_message_input_padding_end` to the `MessageInputFiledView`s `messageInputField` .

### 🎨 UI Changes

_Add relevant screenshots_

| Before | After |
| --- | --- |
| ![Screenshot_20220425-135516_Chat Sample Android](https://user-images.githubusercontent.com/38032787/165084230-b5c20617-57d4-4fc2-a386-f71f3f47e950.jpg) | ![Screenshot_20220425-135439_Chat Sample Android](https://user-images.githubusercontent.com/38032787/165084283-9a2b6503-3b06-4ea7-b256-5b16c75563a1.jpg)
 |

### 🧪 Testing

```
Index: stream-chat-android-ui-components-sample/src/main/res/values/dimens.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/stream-chat-android-ui-components-sample/src/main/res/values/dimens.xml b/stream-chat-android-ui-components-sample/src/main/res/values/dimens.xml
--- a/stream-chat-android-ui-components-sample/src/main/res/values/dimens.xml	(revision 9ad5ef8f6ff4b01b7047279a3cce2df9ae1b053e)
+++ b/stream-chat-android-ui-components-sample/src/main/res/values/dimens.xml	(date 1650887542673)
@@ -10,4 +10,7 @@
 
     <dimen name="badge_horizontal_offset">8dp</dimen>
     <dimen name="badge_vertical_offset">6dp</dimen>
+
+    <dimen name="stream_ui_message_input_padding_end">40dp</dimen>
+    <dimen name="stream_ui_message_input_padding_start">40dp</dimen>
 </resources>

```

1. Run the app on this branch
2. Enter the messages screen, check out the padding on message input text
3. Apply patch above, and repeat

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![little-giants-padding](https://user-images.githubusercontent.com/38032787/165084610-522fe43a-7015-4f4d-95d3-878ba0a7d23e.gif)
